src/sentry/api/event_search.py

```diff
@@ -174,10 +174,21 @@ def visit_date_formats(self, node, children):
 
     def visit_basic_filter(self, node, children):
         search_key, _, search_value = children
+        operator = '='
+
+        # check if there is wildcard syntax
+        if len(search_value) > 1 and '*' in {search_value[0], search_value[-1]}:
+            operator = 'LIKE'
+            chars = list(search_value)
+            for i in (0, -1):
+                if chars[i] == '*':
+                    chars[i] = '%'
+            search_value = ''.join(chars)
```

Comment:
Is it ok if a user sends a value like `*the%thing*`? Should we disallow or escape `%` in user data?

Reply:
changed the way i'm doing this. i don't think this will be a problem because `translate` escapes special characters